grammer.md

```diff
@@ -0,0 +1,73 @@
+# The Grammer as is currently implemented
+```
+Program                             ClassDeclaration*
+ClassDeclaration                    class IDENT { ClassMember* }
+ClassMember                         Field
+                                    | Method
+Field                               public Type IDENT ;
+// TODO: MainMethod missing
```

Comment:
We should differ between main/static methods and normal methods already in the parser/AST. If we find a `static` keyword we just continue in a `parse_main_method`-function. In the AST this will then be expressed by a `ClassMemberKind` enum.

Reply:
I now did a psuedo-ast to explicitly check for this, which we should keep until we get a real AST. I discussed this with @problame. We considered it more elegant to treat MainMethod as special Methods. Same for NewObjectExpression. See 0dc442f